var/spack/repos/builtin/packages/openldap/package.py

```diff
@@ -30,6 +30,7 @@ class Openldap(AutotoolsPackage):
             values=('gnutls', 'openssl'), multi=False)
 
     variant('perl', default=False, description='Perl backend to Slapd')
+    variant('static', default=False)
```

Comment:
Can we avoid the variant and generate both static and shared?

Reply:
Yes, that is possible